go-apps/meep-loc-serv/server/loc-serv.go

```diff
@@ -1233,6 +1239,13 @@ func cleanUp() {
 	userSubscriptionMap = map[int]string{}
 
 	zoneStatusSubscriptionMap = map[int]*ZoneStatusCheck{}
+
+	updateStoreName("")
+}
+
+func updateStoreName(storeName string) {
+	currentStoreName = storeName
+	_ = httpLog.ReInit(logModuleLocServ, sandboxName, storeName, redisAddr, influxAddr)
```

Comment:
This will be called on every active scenario model update, which could happen very often. You should check if the store name changed before calling the httpLog reinit function.

Reply:
true .. updated for both loc-serv and rnis